frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
```

Comment:
If i don't use a global variable i'll need to constantly connect and pull from the database to create the object everytime, that seems wrong and a global variable would suit this perfectly. How would this be bad? or how would you rectify this?